### ISSUE TITLE
teiPrefilter.xsl creates attributes named "xtf-meta" instead of "xtf:meta"

### ISSUE THREAD

**Issue by Conal-Tuohy** [cdlib/xtf#12]
https://github.com/cdlib/xtf/search?utf8=%E2%9C%93&q=xtf-meta

The attributes named `xtf-meta` generated here would therefore be later ignored in [org.cdlib.xtf.textIndexer.XMLTextProcessor.java](https://github.com/cdlib/xtf/blob/61dfd48842465d9c1e2c6dcb36a1c4b1e593a7cf/WEB-INF/src/org/cdlib/xtf/textIndexer/XMLTextProcessor.java#L1477)

**Comment by martinhaye** [comment]
Sorry Conal, I neglected this. Did you close it just because it's been a long time? Should I re-open and merge?

**Comment by Conal-Tuohy** [comment]
Yes please do! Sorry I thought it *had* been merged because I'd just misread the purple "Merged" sign which I now realise was attached to a different PR.
